### DISCUSSION TITLE
nicegui wikidata entry

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [zauberzeug/nicegui#2042] [Show and tell]
Just created the [nicegui wikidata item Q123475332](https://www.wikidata.org/wiki/Q123475332) which is a prerequiste for wikipedia entries for this software. It would be great if the nicegui logo svg would be uploaded to wikimedia commons to properly reference it. This upload must be authorized by the license owner of the logo.

**Comment by rodja** [comment]
Thanks! The Logo is now available at https://commons.wikimedia.org/wiki/File:Logo_nicegui_black.svg

**Reply by falkoschindler** [reply]
I just updated the label from "nicegui" to "NiceGUI".

**Comment by WolfgangFahl** [comment]
Great - the Wikipedia article https://en.wikipedia.org/wiki/NiceGUI is now in the works ...

**Comment by WolfgangFahl** [comment]
at https://en.wikipedia.org/wiki/Talk:NiceGUI the content of the page may be discussed
